README.md

```diff
@@ -46,6 +46,9 @@ Instructions on setting up the development environment and working with the code
 From the `server` directory:
 - `./gradlew run` to start the server
 - `./gradlew test` to test the server
+- `./gradlew checkstyleMain` to run Checkstyle on the server Java code in the `src/main` folder
```

Comment:
I added examples of how to run Checkstyle on the CLI via Gradle. Thanks to @helloworld12321 for the suggestion.